components/OverveiwComponents/limitations.tsx

```diff
@@ -11,12 +11,49 @@ export const Limitations = () => {
         Limitations
       </h1>
       <ul className="list-disc pl-5 space-y-2 md:text-[1.25rem] text-[1rem] text-foreground lg:w-[50rem]">
-        <li>Currently, only the power sector is currently considered.</li>
-        <li>The distribution grid has not yet been modeled in detail.</li>
-        <li>No manual corrections have been applied to input datasets.</li>
         <li>
-          The visualisation is currently available for only 10 countries, with
-          global coverage still in progress.
+          <strong>Sector scope is limited:</strong> Only the power sector is
+          modeled, overlooking transport, industry, and heating—leading to
+          incomplete decarbonization pathways.
+        </li>
+        <li>
+          <strong>No cross-border modeling:</strong> Countries are treated in
+          isolation, missing the benefits of international energy exchange like
+          load balancing and shared infrastructure.
+        </li>
+        <li>
+          <strong>Only single weather-year:</strong> The model uses a single
+          weather year, which may leading to over- or underestimation of
+          generation and storage needs.
+        </li>
+        <li>
+          <strong>Data quality varies globally:</strong> Inconsistent or
+          unvalidated input data can result in unreliable outputs, especially in
+          data-sparse regions.
+        </li>
+        <li>
+          <strong>Limited visualization coverage:</strong> Currently supports
+          only 10 countries, restricting global insights and stakeholder
+          engagement.
+        </li>
+        <li>
+          <strong>Cumbersome data extraction:</strong> Output data is not easily
+          accessible, making detailed analysis and validation more difficult.
+        </li>
+        <li>
+          <strong>Low reproducibility:</strong> Differences in data versions,
+          environment setup, or configuration hinder consistent replication of
+          results.
+        </li>
+        <li>
+          <strong>Single-solution results:</strong> Optimization returns one
+          feasible pathway, missing the broader solution space needed for robust
+          decision-making.
+        </li>
+        <li>
+          <strong>PyPSA-Earth improvements:</strong> The model framework
+          requires general bug fixes, feature and and modularity improvements,
+          as well as documenation updates.
```

Comment:
The fragment sounds like the model is in a really bad state which is 1) not true; 2) not really constructive to attract contributors. I'd suggest rather to mention the distribution grids which modeling is currently one of the major limitations and where we have quite some work ongoing:
```suggestion
          <strong>Overlooking Distribution Grids:</strong> Transmission grid only considered
          which may lead to underestimating the costs and efforts
```